src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.ci

```diff
@@ -32,19 +33,27 @@ module NDMeshStreamer {
     entry void receiveAtDestination(MeshStreamerMessageV *msg);
   };
 
+  template<class dtype, class RouterType>
+  nodegroup MeshStreamerNG {
+    entry MeshStreamerNG();
+    entry [immediate] void setGroupProxy(CProxy_MeshStreamer<dtype, RouterType> groupProxy);
```

Comment:
This entry method had to be marked `immediate` to prevent `MeshStreamerNG::receiveAlongRoute` from getting invoked by `MeshStreamer::sendMeshStreamerMessage` before the group proxy is stored in the nodegroup. Although `MeshStreamerNG::setGroupProxy` is invoked right after the nodegroup is created via `ckNew` (see line 1207 of `xi-Entry.C`), without `immediate` it is not ensured that this entry method is executed before other methods of the nodegroup (e.g. `receiveAlongRoute`).